### PR TITLE
Propose design system name change

### PR DESCRIPTION
**Pull request by elisechant**

### CODE REVIEW

**Comment by dominikwilkowski** [comment]
I fully support this :)

**Review by svict4 — APPROVED** [review]
+1 for GOLD, I still love it.

I do want to ensure that we have "formally the Australian Government Design System" somewhere on the website and in the docs. For any prospective public servant, it'll be crystal clear that this is the system to consider.

File: RFC-by-stage/1-approved/rename-system.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+This would require:
+
+- to update all references of "Australian Government Design System" to "GOLD Design System"
+- to update of all references of "AuDS" to "GOLD"
+- Other branding updates
+- A Press Release or Blog post indicating this has occurred
+
+This would **not** require:
+
+- Change to the npm handle or Github org, as the organisation would be the same
```
Comment (svict4) [inline RFC-by-stage/1-approved/rename-system.md:33]: So to be clear, will we be on the @designsystemau scope? Will we need a one pager on migration steps?

Comment (dominikwilkowski) [reply RFC-by-stage/1-approved/rename-system.md:33]: We should make it clear how to migrate yes. But it should be as simple as this inside the `package.json`:
```diff
 "dependencies": {
-    "@gov.au/accordion": "^3.0.0",
-    "@gov.au/animate": "^3.0.0",
-    "@gov.au/body": "^3.0.0",
-    "@gov.au/breadcrumbs": "^3.0.0",
-    "@gov.au/buttons": "^3.0.0",
+    "@designsystemau/accordion": "^3.0.0",
+    "@designsystemau/animate": "^3.0.0",
+    "@designsystemau/body": "^3.0.0",
+    "@designsystemau/breadcrumbs": "^3.0.0",
+    "@designsystemau/buttons": "^3.0.0",
}
```

Comment (elisechant) [reply RFC-by-stage/1-approved/rename-system.md:33]: We have the scope `@designsystem.au`. I just pinged #feature-contin.. channel, do we potentially want to use another scope `@gold` for instance, instead?

**Review by liamfiddler — APPROVED** [review]
Sensible arguments and reasoning. Let's do this!

**Comment by JordanHatch** [comment]
Fully support this 👍

File: RFC-by-stage/1-approved/rename-system.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## A short description of the desired outcome or solution
+
+This would require:
+
+- to update all references of "Australian Government Design System" to "GOLD Design System"
+- to update of all references of "AuDS" to "GOLD"
+- Other branding updates
+- A Press Release or Blog post indicating this has occurred
+- Rename npm handle to `@gold.au`
```
Comment (elisechant) [inline RFC-by-stage/1-approved/rename-system.md:30]: @dominikwilkowski can you please confirm that we have this handle?

Comment (dominikwilkowski) [reply RFC-by-stage/1-approved/rename-system.md:30]: confirmed

**Review by thesocialdeck — APPROVED** [review]
I agree with these changes. Noting that I think it is important to ensure it is clear that this was formally the Australian Government Design System developed by the DTA. Mainly this is to help fast track approval of sites utilising the system by APS and other government agencies.

**Comment by elisechant** [comment]
This has been approved async ✅  and there are no objections.

**Review by dominikwilkowski — APPROVED** [review]

**Pull request merged**